AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -140,12 +139,51 @@ - (ASCellNodeBlock)tableView:(ASTableView *)tableView nodeBlockForRowAtIndexPath
 
 @end
 
+@interface ASTableViewFilledDataSourceWithRowSize : ASTableViewFilledDataSource
```

Comment:
As you moved the methods to the delegate can you also change all of the names in the test from delegate to dataSource please.